generate/templates/couchbase-server/Dockerfile.template

```diff
@@ -1,3 +1,12 @@
+FROM gcc:9.5.0 as runit_build
+
+RUN set -x \
+    && mkdir /tmp/work \
+    && cd /tmp/work \
+    && git clone https://github.com/couchbasedeps/runit
```

Comment:
Please also take a look at https://github.com/couchbasedeps/runit/commit/edb631449d89d5b452a5992c6ffaa1e384fea697 where I modified runsvdir to handle signals differently.